types/geoip-lite/index.d.ts

```diff
@@ -33,9 +34,20 @@ declare namespace mod {
         area: number;
     }
 
-    export function lookup(ip: string): null | Lookup;
-    export function pretty(ip: number): string;
-    export function startWatchingDataUpdate(): void;
-    export function stopWatchingDataUpdate(): void;
+    interface AsyncCallback {
+        (err?: Error): void;
+    }
+
+    export type CmpArgs = number | [number];
+    export type CmpResult = 1 | -1 | 0 | null;
+
+    export function cmp(a: CmpArgs, b: CmpArgs): null | Lookup;
+    export function lookup(ip: string | number): null | Lookup;
+    export function pretty(ip: string | number | [string | number]): string;
```

Comment:
```suggestion
    export function pretty(ip: string | number | Array<string | number>): string;
```

(`[string | number]` is a singleton of string or number, not an array of string or number)

Reply:
yes, thanks